tests/conftest.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import importlib
+import sys
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture(scope='session')
+def serialzy_location():
+    return Path(__file__).parent.parent
+
+
+@pytest.fixture(autouse=True, scope='session')
+def add_tests_to_sys_modules(serialzy_location):
+    old_sys_path = sys.path.copy()
+    sys.path.append(str(serialzy_location))
+    importlib.import_module('tests')
+    sys.path = old_sys_path
+
+
+def pytest_addoption(parser):
+    parser.addoption("--empty-venv-tests", action='store_true', default=False)
+
+
+def pytest_collection_modifyitems(config, items):
+    empty_venv_tests = config.option.empty_venv_tests
+    serialzy_location = Path(__file__).parent.parent
+
+    for item in items[:]:
```

Comment:
why not just `items`?

Reply:
Because deleting elements of array while iterating over it is not a good idea)